src/roswire/definitions/package.py

```diff
@@ -114,6 +114,34 @@ def paths(shell: dockerblade.Shell,
             paths.extend(package_dirs)
         return paths
 
+    @staticmethod
+    def _paths_ros2(shell: dockerblade.Shell,
+                    files: dockerblade.FileSystem
+                    ) -> List[str]:
+        """Returns paths of packages"""
+        paths: List[str] = []
+        distro = shell.environ('ROS_DISTRO')
+        package_names = '/opt/ros/' + distro + '/share' \
```

Comment:
We're almost there. Just a couple of issues to deal with:
(1) ROS is assumed to be installed to a specific location, and, more importantly, (2) packages within the colcon workspace (e.g., `/ros_ws`) are ignored.